docs/readthedocs/source/doc/Orca/Tutorial/yarn.md

```diff
@@ -43,6 +43,24 @@ The difference between yarn-client mode and yarn-cluster mode is where you run y
 
 For yarn-client, the Spark driver runs in the client process, and the application master is only used for requesting resources from YARN, while for yarn-cluster the Spark driver runs inside an application master process which is managed by YARN in the cluster.
 
+To see the train results, you can look into the driver logs. For **yarn-client** mode, you can directly find the driver logs in the console. For **yarn-cluster** mode, an `application_time_id` will be returned (`application_1668477395550_1045` in the following log) when the application master process is completed.
+
+```bash
+23/02/15 15:30:26 INFO yarn.Client: Application report for application_1668477395550_1045 (state: FINISHED)
+23/02/15 15:30:26 INFO yarn.Client:
+         client token: N/A
+         diagnostics: N/A
+         ApplicationMaster host: ...
+         ApplicationMaster RPC port: 46652
+         queue: ...
+         start time: 1676446090408
+         final status: SUCCEEDED
+         tracking URL: http://.../application_1668477395550_1045/
+         user: ...
+```
+
+Visit the tracking URL and then click `logs` in the table `ApplicationMaster`, where you can see the train results at the end of `stdout` log.
```

Comment:
Visit the tracking URL and then click `logs` in the table `ApplicationMaster` to see the driver logs.